<?xml version="1.0"?>
<launch>
    <arg name="veh" doc="Name of vehicle. ex: megaman"/>
    <arg name="param_file_name" default="default" doc="Specify a param file. ex:default." />
    <arg name="robot_type" default="$(env ROBOT_TYPE)"
         doc="The type of the robot (e.g., duckiebot, watchtower, traffic_light, ...)"/>
    <arg name="robot_configuration" default="$(env ROBOT_CONFIGURATION)"
         doc="The configuration of the robot (e.g., DB19, DB20, WT18, ...)"/>

    <!-- ROS HTTP API: provides an HTTP API through which we can monitor the ROS network -->

    <arg name="/camera/rect" default="true"/>
     <group if="$(arg /camera/rect)">
         <remap from="bot_camera/connection_status" to="connection_status"/>     
         <remap from="bot_camera/car_cmd" to="lane_controller_node/car_cmd"/>
         <remap from="bot_camera/image_in" to="camera_node/image/compressed"/>
         <remap from="bot_camera/parking_start" to="parking_start"/>
         <remap from="bot_camera/image_out" to="camera_node/image/test"/>
        <include file="$(find bot_camera)/launch/test.launch">
            <arg name="veh" value="$(arg veh)"/>
        </include>
    </group>

</launch>
